server/etcdserver/api/v3rpc/grpc.go

```diff
@@ -67,8 +67,7 @@ func Server(s *etcdserver.EtcdServer, tls *tls.Config, interceptor grpc.UnarySer
 	}
 
 	if s.Cfg.EnableDistributedTracing {
-		chainUnaryInterceptors = append(chainUnaryInterceptors, otelgrpc.UnaryServerInterceptor(s.Cfg.TracerOptions...))
-		chainStreamInterceptors = append(chainStreamInterceptors, otelgrpc.StreamServerInterceptor(s.Cfg.TracerOptions...))
+		opts = append(opts, grpc.StatsHandler(otelgrpc.NewServerHandler(s.Cfg.TracerOptions...)))
```

Comment:
Thanks @dims 

The change looks good. But the integrtion test case `TestTracing` only covers Unary rpc, not including stream rpc. It would be great if we can extend the test case to support both Unary and Stream RPC.

To clarify, the [StreamServerInterceptor](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/39ab48e4758bd85e3624efbff4bdb139c9d9f1d1/instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go#L249-L253) has also been deprecated although it hasn't been removed yet.

Reply:
i took a stab at a test case. please see if it looks right. thanks @ahrtr